### DISCUSSION TITLE
Allow mods and scripting in ToonSquid with Lua/Python/etc.

### DISCUSSION THREAD

**Discussion by Lioncub901** [keiwando/toonsquid#552] [Ideas]
It would be cool if we could make mods for ToonSquid. It would also be cool if we could make game with scripting. Ok maybe not games but it would be cool if we could have variables. to make timers or change color at time or position of object . I know this is way too much to ask for but it would be cool.

**Comment by Lioncub901** [comment]
You could use sol2 if you are using lua.

Also having access to position, scale, points, time, and more.

**Reply by kavlor2** [reply]
Made similar suggestion myself. It would be great for extending the effects and tools.

It might be down to Apples approval for something like this though.

**Reply by Lioncub901** [reply]
I feel that Apple would accept it as there is an app called Codea which allows you to make games with lua.
